packages/NodeTypeResolver/Node/AttributeKey.php

```diff
@@ -192,4 +192,9 @@ final class AttributeKey
      * @var string
      */
     public const NEWLINED_ARRAY_PRINT = 'newlined_array_print';
+
+    /**
+     * @var string
+     */
+    public const ASSIGNED_TO = 'assigned_to';
```

Comment:
@TomasVotruba I just re-check that `assigned_to` attribute came from `symplify/phpstan-rules`, which is a `require-dev`, on scoped build, it is not included, so the `AssignedToPropertyNodeVisitor` need to be copied to rector-src

https://github.com/symplify/phpstan-rules/blob/main/src/NodeVisitor/AssignedToPropertyNodeVisitor.php#L15

Reply:
https://github.com/rectorphp/rector-src/pull/3694